README.md

```diff
@@ -4,198 +4,114 @@
-            S3Object getResponse = s3Client.getObject(getRequest);
-            InputStream myObjectBytes = getResponse.getObjectContent();
-
-            // Do what you want with the object
-            
-            myObjectBytes.close();
-
-            return null;
-        }
+Ensure that the `libs` directory is included in your module-level `build.gradle`, under the `dependencies` block:
+```groovy
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
 }
 ```
 
-### SDK Fundamentals
-There are only a few fundamentals that are helpful to know when developing against the AWS SDK for Android.
+## Available Modules
+ 
+ * apigateway-core
+ * auth-core
+ * auth-facebook
+ * auth-google
+ * auth-ui
+ * auth-userpools
+ * autoscaling
+ * cloudwatch
+ * cognito
+ * cognitoauth
+ * cognitoidentityprovider
+ * comprehend
+ * connect
+ * connectparticipant
+ * core
+ * ddb
+ * ddb-document
+ * ddb-mapper
+ * ec2
+ * elb
+ * iot
+ * kinesis
+ * kinesisvideo
+ * kinesisvideo-archivedmedia
+ * kinesisvideo-signaling
+ * kms
+ * lambda
+ * lex
+ * logs
+ * machinelearning
+ * mobile-client
+ * mobileanalytics
+ * pinpoint
+ * polly
+ * rekognition
+ * s3
+ * sagemaker-runtime
+ * sdb
+ * ses
+ * sns
+ * sqs
+ * testutils
+ * textract
+ * transcribe
+ * translate
+
+## SDK Fundamentals
+There are a few fundamentals that are helpful to know when developing against the AWS SDK for Android.
 
 * Never embed credentials in an Android application.  It is trivially easy to decompile applications and steal embedded credentials.  Always use temporarily vended credentials from services such as Amazon Cognito Identity.
 * Unless explicitly stated, calls are synchronous and must be taken off of the main thread.
 * Unless explicitly stated, calls can always throw an AmazonServiceException or an AmazonClientException (depending on if the exception is generated by the service or the client respectively).
 * The SDK will handle re-trying requests automatically, but unless explicitly stated will throw an exception if it cannot contact AWS.
 * We are always looking to help, please feel free to open an [issue](https://github.com/aws-amplify/aws-sdk-android/issues).
 
-## Building the SDK
-This section describes how you can compile the SDK source code on your own.
-
-### Pre-requisites to build the SDK
-
-1. The AWS Core Runtime (`aws-android-sdk-core`) module requires Android API 23 only for compilation locally. Please download and install Android API Level 23 through SDK Manager in Android Studio. 
-
-2. When building the [Core module](https://github.com/aws-amplify/aws-sdk-android/blob/master/aws-android-sdk-core/build.gradle#L32), it requires the following:
-    1. Environment variable `ANDROID_HOME` to be set in order to find the `android-23.jar`.
-    1. Environment variable `ANDROID_PLATFORM` to be set to the platform number (`10` through `28` and above)
```

Comment:
Is this no longer relevant? I remember these steps being essential to building the SDK.

Reply:
NEVERMIND DAVID IS FAST